app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,22 @@
+class CommentsController < ApplicationController
+  before_action :authenticate_with_http_digest
+
+  def create
+    @post = Post.find(params[:post_id])
+    @comment = @post.comments.build(comment_params)
+    @comment.user = current_user
+
+    if @comment.save
+      redirect_to user_post_path(@post.author, @post), notice: 'Bravo, you have created your comment!'
+    else
+      flash[:alert] = 'Apologies try again!'
+      redirect_to user_post_path(@post.author, @post)
+    end
+  end
+
+  private
+
+  def comment_params
+    params.require(:comment).permit(:text)
+  end
+end
```

Comment:
- [ ] [REQUIRED] Kindly note that to properly achieve the goal of this milestone, you need to implement both the `new` and `create` actions for comments just as you did for the post. The new action is typically responsible for rendering the form to create a new comment. The student can create a new comment object in this action and use it in the form.
Something like this:
``` ruby
def new
  @comment = Comment.new
end
```
- Then you will go ahead to Render the Form in the `views/comments/new.html.erb` file which accepts input for creating comments just as you did for Posts
- Make sure that the route for the new action is defined in the config/routes.rb file.